frontend/components/Landing/AboutMeetup.vue

```diff
@@ -21,25 +21,27 @@
       </div>
     </div>
     <div class="relative px-4 sm:px-6 lg:px-8">
-      <div class="mx-auto max-w-prose text-lg">
+      <div class="mx-auto max-w-5xl text-lg m-20 mt-10 ">
```

Comment:
Do you use [eslint](https://eslint.org/) in that project? If no, nice will be if u add him into project (in new `PR`/`issue` - I think)

Reply:
haven't installed it yet, i will do it in next pr :)